pkgs/top-level/python-packages.nix

```
@@ -9505,6 +9513,8 @@ in {
 
   rawkit = callPackage ../development/python-modules/rawkit { };
 
+  ray-bin = callPackage ../development/python-modules/ray/bin.nix { };
```

```suggestion
  ray = callPackage ../development/python-modules/ray { };
```
We don't need a bin and none bin variant and it is not allowed in python packages anyway. As soon as we build from source this is going to be dropped.